presto-main/src/main/java/com/facebook/presto/server/HttpRequestSessionContext.java

```diff
@@ -180,6 +182,7 @@ else if (nameParts.size() == 2) {
             }
         }
         this.systemProperties = systemProperties.build();
+        this.sessionPropertyManager = sessionPropertyManager;
```

Comment:
Move this after tracer's assignment

Reply:
We need this assignment before tracer's to tell if tracing is enabled.

Reply:
Then let's
- move the declaration of `private final Optional<SessionPropertyManager> sessionPropertyManager;` above `tracer`. 
- move `sessionPropertyManager` also right above `tracer`

Overall, the assignment in constructor should follow the same order as the declaration on stack

Reply:
I can move this closer to tracer though